README.md

```diff
@@ -1 +1,23 @@
-# miniCalculator
\ No newline at end of file
+# miniCalculator
+
+This project involved creating a minicalculator in Java using tutorials, research on useful commands, and practice with different classes.
```

Comment:
Could you change this sentence like this for more clarity:
This project involved creating a minicalculator using Java and the librairie Swing.